examples/in-network-rates/in-network-rates-no-npi.json

```diff
@@ -0,0 +1,33 @@
+{
+  "reporting_entity_name": "medicare",
+  "reporting_entity_type": "medicare",
+  "plan_name": "medicaid",
+  "plan_id_type": "hios",
+  "plan_id": "1111111111",
+  "plan_market_type": "individual",
+  "last_updated_on": "2020-08-27",
+  "version": "1.0.0",
```

Comment:
@shaselton-usds , should version be updated in this example?  Or is a 0 NPI usage supported in Version 1.0.0?

Reply:
@ryan-thomas-peterson good observation. I can update the examples in the next PR to Develop.